### PR TITLE
router for GET data from DB is done, INFO now appends on dom

### PR DESCRIPTION
**Pull request by JoshuaDavidOren**
this should be everything for the GET route data appends on my server/ client.

### CODE REVIEW

File: server/routes/koala.router.js [hunk 1]
```diff
@@ -23,6 +23,20 @@ pool.on('error', (err, client) => {
 });
 
 // GET
+koalaRouter.get('/', (req, res) => {
```
Comment (Lycidas-BC) [inline server/routes/koala.router.js:26]: I think the route should match the url in ajax -  '/koalas'

File: server/routes/koala.router.js [hunk 2]
```diff
@@ -23,6 +23,20 @@ pool.on('error', (err, client) => {
...
+
+    let qText = 'Select * FROM "koalalist";';
```
Comment (Lycidas-BC) [inline server/routes/koala.router.js:28]: your query for creating the table has a capital L: koalaList

**Review by Lycidas-BC — COMMENTED** [review]
Looks good! I added two comments - take a look, see if you agree, and then I think you're good to merge.

File: server/routes/koala.router.js (hunk 1 above)
Comment (JoshuaDavidOren) [reply server/routes/koala.router.js:26]: the router uses /koalas to send the get request to the koala.routers.js

File: server/routes/koala.router.js (hunk 2 above)
Comment (JoshuaDavidOren) [reply server/routes/koala.router.js:28]: the table in SQL doesn't respond to the capitalization, I had to debug this

**Pull request merged**